### PR TITLE
Support SetVideoEncoderConfiguration for media ver20

### PR DESCRIPTION
**Pull request by waranlogesh**

### CODE REVIEW

File: lib/media/ver10/profile/multicast_configuration.ex [hunk 1]
```diff
@@ -15,7 +15,7 @@ defmodule Onvif.Media.Ver10.Profile.MulticastConfiguration do
 
     embeds_one :ip_address, IpAddress, primary_key: false, on_replace: :update do
       field(:type, Ecto.Enum, values: [ipv4: "IPv4", ipv6: "IPv6"])
-      field(:ipv4_address, :string)
+      field(:ipv4_address, :string, default: "0.0.0.0")
```
Comment (waranlogesh) [inline lib/media/ver10/profile/multicast_configuration.ex:18]: `nil` was erroring out with invalid argument

File: lib/media/ver20/set_video_encoder_configuration.ex [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    element(:"tt:Multicast", [
+      element(:"tt:Address", [
+        element(
+          :"tt:Type",
+          Keyword.fetch!(
+            Ecto.Enum.mappings(multicast_configuration.ip_address.__struct__, :type),
+            multicast_configuration.ip_address.type
+          )
+        ),
+        element(:"tt:IPv4Address", multicast_configuration.ipv4_address)
```
Comment (hammeraj) [inline lib/media/ver20/set_video_encoder_configuration.ex:63]: as this is the only difference, it's a bit more readable to keep the entire element in the request body and have your pattern match as
```
defp ip_address_element(%{ip_address: %{type: :ipv4}} = multicast_configuration), do: element(:"tt:IPv4Address", multicast_configuration.ipv4_address)

defp ip_address_element(multicast_configuration), do: element(:"tt:IPv6Address", multicast_configuration.ip_address.ipv6_address)
```

File: lib/media/ver10/profile/multicast_configuration.ex (hunk 1 above)
Comment (yuriploc) [reply lib/media/ver10/profile/multicast_configuration.ex:18]: Important to note that some implementations also support the entire `tt:Address` == 0

>  (if this address is set to 0 no multicast streaming is enaled)

(typo by the docs)

Comment (hammeraj) [reply lib/media/ver10/profile/multicast_configuration.ex:18]: as a string or integer?

Comment (waranlogesh) [reply lib/media/ver10/profile/multicast_configuration.ex:18]: as a integer

File: lib/media/ver20/set_video_encoder_configuration.ex (hunk 2 above)
Comment (waranlogesh) [reply lib/media/ver20/set_video_encoder_configuration.ex]: Done

**Review by hammeraj — APPROVED** [review]

**Pull request merged**